script/Arcueid/Day 7/07_00_ARC07_1B.txt

```diff
@@ -167,17 +189,23 @@ I get the feeling that Noel-sensei's education policy is just a tad too lenient.
 [sha:a403179922fcaa025f8be263f733a444984546c9]{
 -- Page 53, Offset 16706.
 -- 「あ、志貴クンったらあたま固いんだ。
-"Oh, Shiki-kun, you're so stubborn.
+// Not sure あたま固い is _stubborn_ in this context, just disapproving of her policy / no fun.
+"Oh, Shiki-kun, you're such a wet blanket.
 }
 [sha:232ce44e27c4ec5ebdd6a8ada1fa02ad49284704]{
 -- Page 53, Offset 16707. Mods: @k, @e.
 -- 先生は不真面目なワケじゃありませーん。生徒の自主性を尊重してのコトですぅー。
-It's not that I'm not serious. It's just that I respect my students' independence.
+// elongated final mora in va translated to !
+It's not that I'm not serious!
+ It's just that I respect my students' independence!
 }
 [sha:38c9797b2f015f4be731f84ff4957452c3420325]{
 -- Page 53, Offset 16708. Glued. Mods: @x.
 -- だってみんな、こんなにも若くて美味しそうなんですもの。一日ぐらい、背伸びして遊びに行くのは仕方がないコトだと思うの」
-I mean, all of you look so young and delicious looking. I think it's inevitable that you guys would go out of your way to play around for a day or so."
+// Leading space for glue
+// 背伸び -> stretch your _legs_ in EN
```

Comment:
Not sure about that. Looks like a [set phrase](https://jisho.org/search/%E8%83%8C%E4%BC%B8%E3%81%B3) closer to "standing on one's tiptoes" or "overreaching beyond one's abilities".